_src/pattern-library/sass/utilities/_helpers.scss

```diff
@@ -122,7 +122,7 @@
     );
 
     border-bottom: 1px solid $transparent;
-    color: palette(primary, base);
+    color: palette(primary, dark);
```

Comment:
Hmm, some thoughts:

* What exact value of gray does this fail on? I'd prefer if we don't clobber all links with a darker color just because of one color combo failure
* by using a dark value, we're completely destroying the nice theming relationship a brand primary value had with links/actions. I'd like to find another way to address this case, please.